IDL/structures.idl

```diff
@@ -162,3 +162,56 @@ struct Structures
     StructBitset var_StructBitset;
     StructEmpty var_StructEmpty;
 };
+
+struct StructShortBIS
+{
+    short var_short;
+};
+
+struct ContentSeqStruct {
+  sequence<StructShort> var_seq;
+};
+
+struct ContentBISSeqStruct {

```

Comment:
I suggest that the BIS is used as a suffix:

```suggestion
struct ContentSeqStructBis {

```

Please apply to any other "Bis" structure